examples/AspNetCore/OData/ODataBasicExample/ODataBasicExample.csproj

```diff
@@ -4,8 +4,4 @@
    <ProjectReference Include="..\..\..\..\src\AspNetCore\OData\src\Asp.Versioning.OData\Asp.Versioning.OData.csproj" />
  </ItemGroup>
 
- <ItemGroup>
-   <PackageReference Update="Microsoft.AspNetCore.OData" Version="9.2.1" />
```

Comment:
Advancing the version used for examples can be current and higher than the minimum version required by the libraries. That version, however, should be set here:

https://github.com/dotnet/aspnet-api-versioning/blob/3fc071913dcded23eeb5ebe55bca44f3828488bf/examples/AspNetCore/OData/Directory.Build.props#L7